<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Databases</class>
 <widget class="QDialog" name="Databases">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open databases</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Ressources.qrc">
    <normaloff>:/Icons/Icons/three128.svg</normaloff>:/Icons/Icons/three128.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>454</width>
        <height>749</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Database&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;connection dialog&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>80</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../Ressources.qrc">:/Icons/Icons/EarthServer.svg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Openned databases</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="databaseactivelabel">
                <property name="text">
                 <string>active:None</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListView" name="listView"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Open a new connection</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <layout class="QFormLayout" name="formLayout">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Server</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="host">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>User</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="user">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>password</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="password">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Database</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="database">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Mysql port</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="mysqlport">
                 <property name="text">
                  <string>3306</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>ssh tunnel</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>ssh options</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_2">
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_6">
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_3">
                          <item>
                           <widget class="QLabel" name="label_7">
                            <property name="text">
                             <string>sshuser</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QLineEdit" name="sshuser">
                            <property name="text">
                             <string/>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_4">
                        <item>
                         <widget class="QLabel" name="label_8">
                          <property name="text">
                           <string>shhserver</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="sshserver">
                          <property name="text">
                           <string/>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_5">
                        <item>
                         <widget class="QLabel" name="label_11">
                          <property name="text">
                           <string>Remote server</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="remotehost"/>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_8">
                      <item>
                       <layout class="QGridLayout" name="gridLayout_6">
                        <item row="0" column="0">
                         <widget class="QLabel" name="label_9">
                          <property name="text">
                           <string>Database server port</string>
                          </property>
                         </widget>
                        </item>
                        <item row="0" column="1">
                         <widget class="QLabel" name="label_10">
                          <property name="text">
                           <string>Local mysql port</string>
                          </property>
                         </widget>
                        </item>
                        <item row="1" column="0">
                         <widget class="QLineEdit" name="databaseserverport">
                          <property name="enabled">
                           <bool>true</bool>
                          </property>
                          <property name="text">
                           <string>3306</string>
                          </property>
                         </widget>
                        </item>
                        <item row="1" column="1">
                         <widget class="QLineEdit" name="localport">
                          <property name="enabled">
                           <bool>true</bool>
                          </property>
                          <property name="text">
                           <string>3306</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QPushButton" name="help">
                      <property name="text">
                       <string>Help </string>
                      </property>
                      <property name="icon">
                       <iconset resource="../Ressources.qrc">
                        <normaloff>:/Icons/Icons/help14.svg</normaloff>:/Icons/Icons/help14.svg</iconset>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="verticalSpacer">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QPushButton" name="SetTunnel">
                      <property name="text">
                       <string>SetTunnel</string>
                      </property>
                      <property name="icon">
                       <iconset resource="../Ressources.qrc">
                        <normaloff>:/Icons/Icons/eye.svg</normaloff>:/Icons/Icons/eye.svg</iconset>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="Previouslogins">
             <property name="text">
              <string>Previous connections</string>
             </property>
             <property name="icon">
              <iconset resource="../Ressources.qrc">
               <normaloff>:/Icons/Icons/handsheart.svg</normaloff>:/Icons/Icons/handsheart.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="okbutton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>connect</string>
             </property>
             <property name="icon">
              <iconset resource="../Ressources.qrc">
               <normaloff>:/Icons/Icons/check46.svg</normaloff>:/Icons/Icons/check46.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Ressources.qrc"/>
 </resources>
 <connections/>
</ui>
